site-docs/src/main/paradox/dynamodb/usage/index.md

```diff
@@ -0,0 +1,40 @@
+# DynamoDB Client
+
+The client constructor takes a `DynamoDbAsyncClient` object. The default `apply` method passes one in with sensible defaults, but you can pass your own in if necessary.
+
+```scala
+val dynamoDbAsyncClient: DynamoDbAsyncClient = ???
+val clientWithCustom = new DADynamoDBClient(dynamoDbAsyncClient)
+
+val clientWithDefault = DADynamoDBClient()
+```
+
+The client exposes two methods:
+
+```scala
+def getAttributeValues(dynamoDbRequest: DynamoDbRequest): F[AttributeValue]
```

Comment:
The method signature doesn't match the signature in the class. The return type is `F[Map[String, AttributeValue]]`